src/components/QuizGameQuestions/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import "./index.css";
+
+// this function will display the fetched data to the dom.those parameters will pass in quizgame component.
+// the callback is used to check if the answer is true or not.
+// types of props ={
+//  question: string
```

Comment:
put spaces after periods. put space after =. there is a colon after question but not after any of the others

Reply:
PTAL